Gemfile

```diff
@@ -15,6 +15,8 @@ gem 'bootstrap-datepicker-rails'
 gem 'autoprefixer-rails', '~> 4.0.2'
 gem 'coffee-rails', '~> 4.1.0'
 gem 'uglifier', '>= 2.6.0' # compressor for JavaScript assets
+# activeadmin dependency
+gem 'inherited_resources', github: 'josevalim/inherited_resources', branch: 'rails-4-2'
```

Comment:
Probably not. My initial intent was to upgrade only the outdated gems highlighted by Gemnasium. Rails just happened as I got carried away.

Also @theB264: AA can work with 4.2, their new lock is < 5.0. However, AA's dependency inherited_resources is not ready yet for 4.2 on master, which is why it's sourced from a feature branch. For what it's worth, there is a disclaimer on the [inherited_resources' repo](https://github.com/josevalim/inherited_resources), presumably from its creator Jose Valim, which reads
> Since Rails 3 came out, I have no longer used Inherited Resources and it is no longer actively maintained.

The repo is somewhat active, though most of that activity appears to be maintenance. So, master branch or feature, we will be/are using a pretty stale dependency.

Let me know if rollback to 4.1.9 is preferred (we do need to upgrade from 4.1.8 to 4.1.9 to enable other gems' updates).